src/rcx/test/BUILD

```diff
@@ -0,0 +1,101 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2022-2025, The OpenROAD Authors
+load("//test:regression.bzl", "regression_test")
+
+# From CMakeLists.txt or_integration_tests(TESTS
+COMPULSORY_TESTS = [
+    "45_gcd",
+    "ext_pattern",
+    "gcd",
+    "generate_pattern",
+    "names",
+]
+
+# Disabled in CMakeLists.txt
+MANUAL_TESTS = [
+    "generate_rules",
+    "rcx_man_tcl_check",
+    "rcx_readme_msgs_check",
+]
+
+ALL_TESTS = COMPULSORY_TESTS + MANUAL_TESTS
+
+filegroup(
+    name = "regression_resources",
+    # Dependencies could be specified more narrowly per test case,
+    # but at least it is not a glob of everything and there are
+    # per test glob patterns below.
+    srcs = [
+        "45_patterns.rules",
+        "45_via_resistance.tcl",
+        "Nangate45/Nangate45.lef",
+        "Nangate45/Nangate45.pdn.tcl",
+        "Nangate45/Nangate45.rc",
+        "Nangate45/Nangate45.rcx_rules",
+        "Nangate45/Nangate45.tracks",
+        "Nangate45/Nangate45.vars",
+        "Nangate45/Nangate45_fast.lib",
+        "Nangate45/Nangate45_lvt.lef",
+        "Nangate45/Nangate45_lvt.lib",
+        "Nangate45/Nangate45_slow.lib",
+        "Nangate45/Nangate45_stdcell.lef",
+        "Nangate45/Nangate45_tech.lef",
+        "Nangate45/Nangate45_typ.lib",
+        "Nangate45/fake_macros.lef",
+        "Nangate45/fake_macros.lib",
+        "Nangate45/fakeram45.cfg",
+        "Nangate45/fakeram45_1024x32.lef",
+        "Nangate45/fakeram45_1024x32.lib",
+        "Nangate45/fakeram45_256x16.lef",
+        "Nangate45/fakeram45_256x16.lib",
+        "Nangate45/fakeram45_512x64.lef",
+        "Nangate45/fakeram45_512x64.lib",
+        "Nangate45/fakeram45_64x32.lef",
+        "Nangate45/fakeram45_64x32.lib",
+        "Nangate45/fakeram45_64x7.lef",
+        "Nangate45/fakeram45_64x7.lib",
+        "Nangate45/fakeram45_64x96.lef",
+        "Nangate45/fakeram45_64x96.lib",
+        "Nangate45/work_around_yosys/cells.v",
+        "helpers.tcl",
+        "patterns.rulesok",
+        "rcx_unit_test.tcl",
+        "rulesGen.log",
```

Comment:
This doesn't belong.  Removed in https://github.com/The-OpenROAD-Project/OpenROAD/pull/7075